avalon/gui/__init__.py

```diff
@@ -0,0 +1,88 @@
+from ..vendor import qtawesome
+from ..vendor.Qt import QtCore, QtWidgets, QtGui
+from .. import io, api, pipeline
+
+from . import style
+
+from . import lib
+
+# ----- Models -----------------------------------------------------
+from .models.node import Node
+
+from .models.model_tree import TreeModel
+from .models.model_task import TaskModel
+from .models.model_asset import AssetModel
+from .models.model_subset import SubsetModel
+from .models.model_inventory import InventoryModel
+
+from .models.proxy_filter import FilterProxyModel
+from .models.proxy_exact_matches_filter import ExactMatchesFilterProxyModel
+from .models.proxy_recursive_sort_filter import RecursiveSortFilterProxyModel
+from .models.proxy_group_filter import GroupMemberFilterProxyModel
+from .models.proxy_subset_filter import SubsetFilterProxyModel
+from .models.proxy_family_filter import FamilyFilterProxyModel
+
+# ----- Delegates --------------------------------------------------
+from .delegates.delegate_pretty_time import PrettyTimeDelegate
+from .delegates.delegate_version import VersionDelegate
+
+# ----- Views -------------------------------------------------------
+from .views.view_tree_deselectable import DeselectableTreeView
+from .views.view_assets import AssetsView
+
+# ----- Widgets -----------------------------------------------------
+from .widgets.widget_subsets import SubsetsWidget
+from .widgets.widget_families_list import FamiliesListWidget
+from .widgets.widget_silos_tab import SilosTabWidget
+from .widgets.widget_assets import AssetsWidget
+from .widgets.widget_version_text_edit import VersionTextEdit
+from .widgets.widget_version import VersionWidget
+
+
+__all__ = [
+    "QtCore",
+    "QtWidgets",
+    "QtGui",
+
+    "qtawesome",
+
+    "api",
+    "io",
+    "pipeline",
```

Comment:
Dont think we need; `QtCore`, `QtWidgets`, `QtGui`, `qtawesome`, `api`, `io`, `pipeline` exposed here.

Reply:
With this is much easier to import them in models, delegates, views and widgets.
e.g. instead of `from ....vendor.Qt import QtWidgets, QtCore` in each file you can just use `from .. import QtWidgets, QtCore`

Reply:
> With this is much easier to import them in models, delegates, views and widgets.
e.g. instead of from ....vendor.Qt import QtWidgets, QtCore in each file you can just use from .. import QtWidgets, QtCore

No, more explicit getting it from vendor will definitely avoid confusion as to where it's coming from. In this case explicit beats exposing it in so many locations. So I say @tokejepsen is right 👍